tests/migration/migration.go

```diff
@@ -920,7 +920,7 @@ var _ = SIGMigrationDescribe("VM Live Migration", func() {
 
 				// Only stressing the VMI for 60 seconds to ensure the first migration eventually succeeds
 				By("Stressing the VMI")
-				runStressTest(vmi, stressDefaultVMSize, 60)
+				runStressTest(vmi, stressDefaultVMSize)
```

Comment:
```
// Only stressing the VMI for 60 seconds to ensure the first migration eventually succeeds
```
I see this comment... Don't we need this anymore? I mean this 60 seconds are not needed anymore?

Reply:
I need to clean the comment, but if you look at the function it always reduced the timeout to 15 second max anyway...

Reply:
lol xD